### DISCUSSION TITLE
3 different octobot run with 1 account

### DISCUSSION THREAD

**Discussion by mertdogan** [Drakkar-Software/OctoBot#1704] [Q&A]
My plan:
I installed octobot at 3 different device. Each device connects same account with different api keys and different octobot strategies. 

Does this structure cause conflict problems about Octobot because of different strategy trade operations and commands?

**Comment by Herklos** [comment]
Hi @mertdogan, sure. By using the same account the different strategies will not be able to work correctly

**Reply by mertdogan** [reply]
I'm sorry now :/ but thank you.

**Comment by GuillaumeDSM** [comment]
If you use different trading pairs, it will be fine though

**Reply by mertdogan** [reply]
Hmm this is very good information; it is very good.

**Comment by mertdogan** [comment]
But i think, if you add a setting to use octobot only base % of user funds (for example ; i have 10BTC and i set octobot to use only %10 of this fund) this will solves my needs without pair depomposition.

**Comment by GuillaumeDSM** [comment]
if you use a strategy that allows to do this like the grid trading mode then you can do it yes

**Comment by BlueCute** [comment]
Can someone tell me good strategy and how to set up correctly please. And what kind of pairs are better to be used now.
Cant find a video with good example i can use. For that i ask here.
I have money in Binance, I also can put money in Kraken or coinbase.
I am sorry i just try to learn.

**Comment by BlueCute** [comment]
2022-09-15 19:56:35,111 WARNING  PortfolioValueHolder[ftx] No trading pair including EUR and USDT on ftx. EUR can't be valued for portfolio and profitability.
 2022-09-15 19:56:41,917 ERROR    WebConfigurationModel Failed to get currencies list from coingecko.com : string indices must be integers

I dont understand .. does not work with euro ?

I don't understand why no one answers.
I bet most here are completely familiar with the bot configuration and everything else.
1-2 people answer and that's it.

Even out of politeness no one bothers to give directions.

**Reply by GuillaumeDSM** [reply]
I dont understand .. does not work with euro ?
=> to give a value to EUR in your reference market, OctoBot need a traded pair to convert EUR to your reference market. It looks like here FTX in not providing any EUR/USDT pair and therefore OctoBot can't say how much USDT is worth 1 EUR in your portfolio. This is just a warning and is only affecting the values in your portfolio tab.

2022-09-15 19:56:41,917 ERROR WebConfigurationModel Failed to get currencies list from coingecko.com : string indices must be integers
=> We changed the way this error is handled in the latest version, you can ignore it (coingecko.com is used to get crypto logos and sometimes it reaches a rate limit then gets back on)

As a note, most people are on OctoBot telegram / discord  groups, you will have faster answers there
